addons/project_todo/models/project_task.py

```diff
@@ -51,4 +51,5 @@ def action_convert_to_task(self):
             'res_model': 'project.task',
             'res_id': self.id,
             'type': 'ir.actions.act_window',
+            'context': {**self._context, 'show_todo_breadcrumb': False},
```

Comment:
you can revert that change, we will not magically remove a item in the breadcrumb